ci/requirements/environment.yml

```diff
@@ -47,3 +48,4 @@ dependencies:
   - zarr
   - pip:
     - numbagg
+    - git+https://github.com/dcherian/dask_groupby.git
```

Comment:
Todo switch away from git repo before merge. `dask_groupby` is on conda-forge now.